src/components/LandingPageComponent.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import image from "../images/imagegrid.png";
+import { Button, Typography } from "@mui/material";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { Grid } from "@material-ui/core";
+import logo from "../images/logo.png";
+const theme = createTheme({
```

Comment:
move createTheme to separate file, you can name it theme.js. You will need to override theme many times and you don't want to use createTheme in every component, separate file is best place to keep it.